### PR TITLE
Move the docs out from the README.md

### PR DESCRIPTION
**Pull request by bocharsky-bw**
@weaverryan idea. Is `docs/index.md` a good place?

### CODE REVIEW

**Comment by weaverryan** [comment]
Hey! Thanks for getting this rolling :). Really, what we need to do is:

A) Move this to a doc/index.rst file
B) Create a `.symfony.bundle.yaml` file
C) Ping Javier to add this to https://symfony.com/bundles

e.g. like https://github.com/symfony/ux/tree/2.x/src/Autocomplete :)

**Comment by bocharsky-bw** [comment]
@weaverryan changes are done! Should this PR be merged first and then we need to ping Javier?

File: README.md [hunk 1]
```diff
@@ -1,347 +1,12 @@
...
-
-You can also retrieve the generated attributes as an array, which can be helpful e.g. for forms:
-
-```twig
-{{ form_row(form.password, { attr: stimulus_target('hello-controller', 'a-target').toArray() }) }}
-```
-
-Ok, have fun!
+[1]: https://symfony.com/doc/current/frontend.html
+[2]: https://github.com/symfony/webpack-encore-bundle/blob/main/doc/index.rst
```
Comment (weaverryan) [inline README.md]:
```suggestion
[2]: https://symfony.com/bundles/WebpackEncoreBundle/current/index.html
```

That doesn't exist yet, but that's where we should take them.

**Review by weaverryan — COMMENTED** [review]
Looks good! Just one tweak. And yes, we'll merge that I'll ping Javier

File: README.md (hunk 1 above)
Comment (bocharsky-bw) [reply README.md]: I was thinking about fixing it after we will have a working URL, but let's do it in advance :)

**Comment by weaverryan** [comment]
Thanks Victor!

**Pull request merged**